indigo_api/views/documents.py

```diff
@@ -356,7 +356,7 @@ def post(self, request, document_id):
             doc.frbr_uri = frbr_uri
             xml = doc.to_xml(encoding='unicode')
 
-        return Response({'output': xml})
+        return Response({'output': xml, 'provisionEid': serializer.validated_data.get('provision_eid')})
```

Comment:
JSON responses usually use the `provision_eid` style member names. It's good to standardise on that rather than having the request and the response use different names.